maat-court-data-api/src/main/java/gov/uk/courtdata/entity/RepOrderCapitalEntity.java

```diff
@@ -0,0 +1,70 @@
+package gov.uk.courtdata.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "REP_ORDER_CAPITAL", schema = "TOGDATA")
+public class RepOrderCapitalEntity {
+
+
+    @Id
+    @Column(name = "ID", nullable = false, precision = 0)
```

Comment:
This is more restrictive. Please update the Entity

Reply:
Thanks. Done